dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/srt/SRTServletResponse.java

```diff
@@ -2252,7 +2252,12 @@ public void addDateHeader(String name, long date) {
                 throw new IllegalArgumentException("Cannot Set Content-Type to a Date value");
             }
             else {
-                setHeader(name, (createCompliantHttpDateString(date)));
+                if (com.ibm.ws.webcontainer.osgi.WebContainer.getServletContainerSpecLevel() < com.ibm.ws.webcontainer.osgi.WebContainer.SPEC_LEVEL_61) {
+                    setHeader(name, (createCompliantHttpDateString(date)));
+                }
+                else {
+                    addDateField(name, date);
+                }
```

Comment:
Let's make sure to add this to the UFO as a Servlet 6.1 behavior difference for Open Liberty

Reply:
Track via ToDo issue
 https://github.com/OpenLiberty/open-liberty/issues/27383